processor/file.go

```diff
@@ -162,24 +139,19 @@ func walkDirectoryParallel(root string, output chan *FileJob) {
 				}
 			}
 
-			if gitIgnoreError == nil && gitIgnore.Match(filepath.Join(root, f.Name()), true) {
-				if Verbose {
-					printWarn("skipping directory due to git ignore: " + filepath.Join(root, f.Name()))
-				}
-				shouldSkip = true
-			}
-
-			if ignoreError == nil && ignore.Match(filepath.Join(root, f.Name()), true) {
-				if Verbose {
-					printWarn("skipping directory due to ignore: " + filepath.Join(root, f.Name()))
+			for _, i := range ignores {
```

Comment:
Given how commonly `i` is used for numeric indexes, I would avoid using it here. `ignore` is clearer and not too long.